tools/serviceTest.sh

```diff
@@ -17,13 +17,19 @@ match=$2
 timeOut=$3
 START=$SECONDS
 
-##################################### Check the operating system #########################################
-if [ $(uname -s) == "Darwin" ]; then
-    # This is a MAC machine
-    command="docker logs -f `docker ps -q --filter name=$name`"
+##################################### Check for hzn service log command and OS ###########################
+logCommand=$( hzn service log 2>&1 )
+if [ "$logCommand" == 'hzn: error: expected command but got "log", try --help' ]; then
```

Comment:
It is best not to depend on the error msg having certain text in it (because that could change). Instead, run the cmd `hzn service log -h`. That returns exit code 0 if it is there and exit code 1 if it isn't.